token-lending/program/src/pyth.rs

```diff
@@ -0,0 +1,148 @@
+#![allow(missing_docs)]
```

Comment:
Is this file copied from somewhere? Could you add a comment about that if so?

Reply:
Done.

Reply:
Done.